loop.py

```diff
@@ -59,6 +60,41 @@ def compute_unfairness(A, R):
     return np.abs(np.asarray(A) - np.asarray(R))
 
 
+def compute_max_unfairness(A, R, k):
+    min_value = sys.maxsize
+    unfairness = (np.asarray(A) - np.asarray(R))[::-1]
+    index = len(unfairness) - 1
+    result = -1
+
+    for item in unfairness:
```

Comment:
is super slow, there optimized utils on in numpy

Reply:
Will have a look.